splitResources.js

```diff
@@ -0,0 +1,51 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const fs = require("fs");
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const path = require("path");
+
+exports.default = async function (buildResult) {
+  const segmentSize = 1 * 1024 ** 3; // 1GB
+  const fileName = "voicevox-0.3.1-x64.nsis.7z"; // targe file name
+  const targetDirectory = path.resolve(buildResult["outDir"], "nsis-web"); // for nsis-web
+  const outputDirectory = path.resolve(targetDirectory, "out");
+  const inputFile = path.resolve(targetDirectory, fileName);
+
+  console.log("Split file.");
+  const inputStream = fs.createReadStream(inputFile, {
+    encoding: null,
+    highWaterMark: segmentSize,
+  });
+
+  let file_index = 0;
```

Comment:
細かいですが、camelCaseが一般的です
```suggestion
  let fileIndex = 0;
```

Reply:
7c12da2 で対応しました。